src/Resources/Guest/index.ts

```diff
@@ -0,0 +1,12 @@
+import Client from '../../Client';
+import Basket from './Basket/Basket';
+
+export interface GuestResources {
+    basket: Basket;
+}
+
+export function loadGuestResources(client: Client): GuestResources {
+    return {
+        basket: new Basket(client),
```

Comment:
Could this be called `baskets`?

Reply:
Can you clarify why it should be plural please? When adding to a guest basket you are referring to one basket, not multiple. That's why I used singular.